docs/core/reports/customisation.md

```diff
@@ -83,6 +83,22 @@ Then this will be displayed at the top of reports:
 
 Note that you can also specify a path to a config file using `-c`.
 
+## Listing software versions
+
+For some modules the software version can be parsed from the log files. If version
+information is not included in the log files, you can instead add them in config.
+Software versions are specified in the YAML as pairs of software names and version
+strings. Multiple version of the same software are specified as lists. Example:
+
+```yaml
+software_versions:
+  samblaster: "0.1.24"
+  samtools:
+    - "1.15"
+    - "1.10"
+  some_other_tool: "2023-1"
+```
+
```

Comment:
hah, sorry 😅  Yes I think it's good to have both 👍🏻

Reply:
I was also thinking mostly on pipelines when writing this so that why I like the idea of a separate YAML file. 

Also one thing with providing version in the configs is that there are multiple scope-levels. If you, for whatever reason, add software version information on a system or user config level, it will appear in all reports.